src/helm/clients/together_client.py

```diff
@@ -272,3 +280,86 @@ def do_it_sync() -> Dict[Any, Any]:
                 completions=completions,
                 embedding=[],
             )
+
+
+class TogetherRawChatRequest(TypedDict):
+    messages: List[Dict[str, str]]
+    model: str
+    max_tokens: int
+    stop: List[str]
+    temperature: float
+    top_p: float
+    top_k: int
+    logprobs: int
+    echo: bool
+    n: int
+
+
+def convert_to_raw_chat_request(request: Request) -> TogetherRawChatRequest:
+    if request.messages:
+        messages = request.messages
+    else:
+        messages = [{"role": "user", "content": request.prompt}]
+    return {
```

Comment:
Do you have to convert the dict into a `TogetherRawChatRequest`?

Reply:
Nope, the dict automatically counts as a `TogetherRawChatRequest` without any explicit conversion needed ([PEP-589](https://peps.python.org/pep-0589/))